datadog_checks_base/tests/test_agent_check.py

```diff
@@ -256,6 +256,13 @@ def test_none_value(self):
         normalized_tags = check._normalize_tags_type(tags, None)
         assert normalized_tags == ['tag:foo']
 
+    def test_none_value_warning_log(self, caplog):
```

Comment:
No a big deal, but should we consider merging `test_none_value` and  `test_none_value_warning_log` into one test ?

e.g.

```python
    def test_none_value(self, caplog):
        check = AgentCheck()
        tags = [None, 'tag:foo']

        normalized_tags = check._normalize_tags_type(tags, None)

        assert normalized_tags == ['tag:foo']
        assert 'Error encoding tag' not in caplog.text
```

Reply:
Yes I think this is a good idea!
Just modified it.

Thanks!